ambari-server/src/main/java/org/apache/ambari/server/controller/DeleteIdentityHandler.java

```diff
@@ -52,6 +55,7 @@
 import org.apache.ambari.server.state.kerberos.KerberosDescriptor;
 import org.apache.ambari.server.state.svccomphost.ServiceComponentHostServerActionEvent;
 import org.apache.ambari.server.utils.StageUtils;
+import org.springframework.util.CollectionUtils;
```

Comment:
Should probably be `org.apache.commons.collections.CollectionUtils`

Reply:
I knew I was going to mess this up.  Fixed other places where my IDE choose the wrong impl. Nice catch.